.travis.yml

```diff
@@ -39,7 +39,7 @@ script:
     then
       bash scripts/test-make-requirements.sh
     fi
-  - "scripts/docker test --noinput --stop --verbosity=2 --divide-depth=1 --with-timing --threshold=10"
+  - "scripts/docker test --noinput --verbosity=2 --divide-depth=1 --with-timing --threshold=10"
```

Comment:
Is removing `--stop` an intentional part of this PR?

Reply:
not permanently